yb-voyager/src/cp/yugabyted/yugabyted.go

```diff
@@ -44,13 +48,54 @@ type YugabyteD struct {
 	latestInvocationSequence int
 }
 
+type VoyagerInstance struct {
```

Comment:
Moved just `VoyagerInstance` to `ControlPlane` pacakge.

Reply:
Ah so there is probably no dependency from cp -> yugabyted packages. 
Ideally, we should try and not have the reverse dependency either. (right now, yugabytedb depends on cp package because of `VoyagerInstance`.  If we use `json.Marshaller`, that dependency also goes away. 

Add a TODO in case you're not picking this up right now

Reply:
in yugabyted.go at multiple places we use the cp package dependency. For example events like ImportSchemaStartEvent, etc...

I didn't get what can't we have that dependency here.
`yugabyted` implements the `controlPlane` interface and it should be fine to refer that in here.

Reply:
Ah I see. Just assumed that since yugabyted was an implementation of a control plane, it wouldn't depend on the control plane package itself. I'll give it another look 👍

Reply:
This makes sense 👍 
ControlPlane package just has the interface and types. 
yugabyted depends on ControlPlane (for the interface and types)
root package depends on controlPlane(and not yugabyted)